cfpb/cfgov-refresh
# Mega Menu: Set header minimum height

[PR 4046](https://github.com/cfpb/cfgov-refresh/pull/4046) changed the behavior of the menu to show on load before the JS kicked in, in order to make the rendering on eregs with turbolinks less janky. However, this introduced a flickr into the loading appearance, since the menu flickers open before being hidden by JS. This PR reverts 4046 and instead sets a minimum height on the header so that the space for the menu is allocated before it loads in, eliminating the jump that happens to the header as it expands in height when the menu has loaded.

## Changes

- Revert PR [#4046](https://github.com/cfpb/cfgov-refresh/pull/4046)
- Set a minimum height on the header at tablet/desktop sizes.

## Testing

1. Pull branch, run `gulp clean && gulp build`
2. Visit any regular page (homepage is fine) and http://localhost:8000/policy-compliance/rulemaking/regulations/ 
3. Open the dev tools, go to the performance tab, and select Slow 3G for the Network, to simulate a slow connection.
4. Reload the page and observe now the page does not jump as the menu loads in

## Notes

- I noticed the first three menu items would sometimes show and the last two would be hidden as it was loading. I'm not sure if this is a by-product of the simulation or if it's really happening.
